depthai_nodes/ml/messages/mapping.py

```diff
@@ -0,0 +1,26 @@
+import depthai as dai
+import numpy as np
+
+
+class Map2D(dai.Buffer):
+    def __init__(self):
+        super().__init__()
+        self._map: np.ndarray = None
+        self._width: int = None
+        self._height: int = None
```

Comment:
We set _width and _height but we never use them. Should we have getters for those two?